src/types/pageURL.ts

```diff
@@ -31,3 +31,9 @@ export const PERSONS_NAME = "/what-is-the-persons-name";
 export const PERSONAL_CODE = "/companies-house-personal-code";
 
 export const EMAIL_ADDRESS = "/what-is-their-email-address";
+
+export const HOME_ADDRESS = "/address-lookup";
+
+export const HOME_ADDRESS_MANUAL = "/address-manual";
+
```

Comment:
The url should be "/enter-their-home-address" as per the ticket

Reply:
Changed